cli/cli/helpers/portal_manager/file_helpers.go

```diff
@@ -0,0 +1,24 @@
+package portal_manager
+
+import (
+	"github.com/kurtosis-tech/stacktrace"
+	"os"
+)
+
+// CreateFileIfNecessary creates the file if it doesn't already exist. If it does, do nothing.
+// Return true if the file was created, false if it existed already, and error if something went wrong
+func CreateFileIfNecessary(filePath string) (bool, error) {
+	if _, err := os.Stat(filePath); err != nil {
```

Comment:
Well, here we have 3 different cases if I'm not mistaken:
- err != nil && IsNotExist == true -> create the file
- err != nil -> return an unexpected error
- err == nil -> file exists already, don't do anything
So, we do have to distinguish between the three anyway. I can refactor this to a `if ... else if ... else`, but can't really simplify the overall logic. I'll do the refactoring as it will have less indentation